### DISCUSSION TITLE
Pending Sample Results?

### DISCUSSION THREAD

**Discussion by lam3234** [opioiddatalab/drugchecking#4] [General]
I have been waiting on a pending sample result - 300510. It has been pending for a long time. Wondering when it might be updated. Sample was sent in near the beginning of April. Thanks so much!

**Comment by ectracy** [comment]
Hi - The results for 300510 should be uploaded to the website today (https://www.streetsafe.supply/) today (6/1/23). I apologize for the delay in sample analysis. The sample was run 3 times however no compounds of interest were detected. This could be because the sample did not contain any active ingredients, the concentration of the compounds was below what the instrument could detect or the sample contained a compound not readily detected by GCMS. Please reach out if you have any further questions! -- Erin

**Reply by Pursuit63** [reply]
Also was Wondering if you had any info on a sample 601552 sent out that last week of October  I was not able to find any info on the sample in the pending or results page .  Any info regarding this would be helpful and appreciated.

**Reply by nabarunDG** [reply]
We just checked and we haven't received it in the lab yet. Maybe check with the staff of the local program? Sorry about that!
